### PR TITLE
Rename Geospatial to Microplanning

### PR DESCRIPTION
**Pull request by zandre-eng**
## Product Description

A new Microplanning Feature Flag toggle has been added to replace the current Geospatial FF:
Screenshot from 2024-11-08 10-33-02

Additionally, all relevant URL slugs and UI references have been updated to correctly name the various Microplanning features as they are being called today:
Screenshot from 2024-11-08 10-33-19

## Technical Summary

Link to ticket [here](https://dimagi.atlassian.net/browse/SC-4020).

This is quite a straightforward PR which mostly consists of UI changes in an effort to transition the Geospatial feature to a Microplanning naming. This involved changing the current Geospatial toggle to Microplanning. Doing so will mean that all current domains will need to be re-added to the new FF. These will be done with a manual migration (see Jira ticket for more context).

## Feature Flag

`GEOSPATIAL` being changed to `MICROPLANNING`

### Safety story

- Local testing
- Mostly UI only changes, apart from toggle rename

### Automated test coverage

N/A

### QA Plan

No QA planned.

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by Charl1996 — APPROVED** [review]
Looks good to me.

**Review by ajeety4 — APPROVED** [review]

File: corehq/toggles/__init__.py [hunk 1]
```diff
@@ -2600,14 +2600,13 @@ def _handle_attendance_tracking_role(domain, is_enabled):
     save_fn=_handle_attendance_tracking_role,
 )
 
-GEOSPATIAL = StaticToggle(
-    'geospatial',
-    'Allows access to GIS functionality',
+MICROPLANNING = StaticToggle(
+    'microplanning',
```
Comment (mkangia) [inline corehq/toggles/__init__.py:2604]: Hey @zandre-eng 

Just a suggestion that you can retain the same feature flag if you retain the same slug which I think isn't bad unless the switch over is easy.

Comment (zandre-eng) [reply corehq/toggles/__init__.py:2604]: Hey @mkangia,

The switchover is fortunately fairly easy. On production there are mostly just test domains using this feature flag and they will simply be manually switched over to the new feature flag toggle once it's live. I think it's a good opportunity to switch it over now while this feature isn't being widely used yet.

**Review by mkangia — APPROVED** [review]
I believe you would want to rename the django app also from geospatial to microplanning.

**Comment by zandre-eng** [comment]
> I believe you would want to rename the django app also from geospatial to microplanning.

@mkangia That would be ideal, along with changing all the code references to "Geospatial". This will definitely be a bigger lift however, and after having a chat with some other folks, I thought it best to handle all the code renamings in a separate PR. I will create a separate Jira ticket for this.

**Pull request merged**